R/api_url_query_utils.R

```diff
@@ -0,0 +1,35 @@
+#' Create <filter>.in query string for URL queries
+#'
+#' @description
+#' Outputs a URL query string containing timePeriods.in=..., geographicLevels.in=..., etc for
+#' use with querying a data set via GET.
+#'
+#' @param items items to be included in the "in" statement
+#' @param type type of filter items being queried: "time_periods", "geographic_levels",
+#' "locations" or "filters"
+#'
+#' @return Query string for use in URL based API queries
+#' @export
+#'
+#' @examples
+#' parse_filter_in(c("2024|W11", "2024|W12"), type = "time_periods")
+parse_filter_in <- function(
```

Comment:
This maybe helps a little, but I've shifted the filter type option to being "filter_items" instead of filters, so that there's a bit of a difference between the use of filter in the function name and the specific type choice of "filter". Was going to do this first time around, but ended up matching to the URL use of filters.in, rather than what I'd used elsewhere in the package.